ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventCaseProg-nonprod.json

```diff
@@ -60,5 +60,23 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "TRIAL_READINESS",
+    "AccessControl": [
+      {
+        "UserRoles": [
```

Comment:
Is this only for solicitors ? Judge / LA does not need access ?

Reply:
[CIV-6051](https://tools.hmcts.net/jira/browse/CIV-6051) in the user story and acceptance criteria only mentions claimant/defendant (LR)

Reply:
I consulted this with Doug, and he agrees.